18c_xe/createUserAndTS.sh

```diff
@@ -4,20 +4,20 @@ $ORACLE_HOME/bin/sqlplus / as sysdba << EOF
 
 startup
 
-CREATE USER secretsysuser
-  IDENTIFIED BY secretpassword
+CREATE USER system
+  IDENTIFIED BY oracle
   DEFAULT TABLESPACE SYSTEM
   TEMPORARY TABLESPACE SYSTEM
   PROFILE DEFAULT
   ACCOUNT UNLOCK;
 
--- 1 Role for secretsysuser 
-GRANT DBA TO secretsysuser;
-ALTER USER secretsysuser DEFAULT ROLE ALL;
+-- 1 Role for system 
+GRANT DBA TO system;
+ALTER USER system DEFAULT ROLE ALL;
 
--- 2 System Privileges for secretsysuser 
-GRANT CREATE SESSION TO secretsysuser;
-GRANT UNLIMITED TABLESPACE TO secretsysuser;
+-- 2 System Privileges for system 
+GRANT CREATE SESSION TO system;
+GRANT UNLIMITED TABLESPACE TO system;
```

Comment:
I've been running into issues with having changed this to `system` possibly because `system` already exists and there might be something special about it. Going to rename this.